Week3/homework/step2-3.js

```diff
@@ -31,8 +35,13 @@ function repeatStringNumTimesWithDoWhile(str, num) {
   // eslint-disable-next-line
   let result = '';
 
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  let i = 0;
+  do {
+    if (num > 0) {
```

Comment:
By placing the `if` statement inside the loop it is executed for each loop iteration. However, it needs to be tested once only, before the loop executes.